config/base/watcher.yaml

```
@@ -36,12 +36,10 @@ spec:
         - name: watcher
           image: ko://github.com/tektoncd/results/cmd/watcher
           args:
-            [
-              "-api_addr",
-              "tekton-results-api-service.tekton-pipelines.svc.cluster.local:8080",
-              "-auth_mode",
-              "token",
-            ]
+            - -api_addr
+            - $(TEKTON_RESULTS_API_SERVICE)
```

Would this be cleaner?
```suggestion
            - -api_addr=$(TEKTON_RESULTS_API_SERVICE)
```

---

Either way seems fine.